crates/dprint/src/environment/real_environment.rs

```diff
@@ -99,8 +99,14 @@ impl Environment for RealEnvironment {
     fn glob(&self, base: &Path, file_patterns: &Vec<String>) -> Result<Vec<PathBuf>, ErrBox> {
         let start_instant = std::time::Instant::now();
         log_verbose!(self, "Globbing: {:?}", file_patterns);
+        let os_file_patterns = if cfg!(windows) {
+            file_patterns.iter().map(|v| v.replace("\\", "/")).collect::<Vec<String>>()
+        } else {
+            file_patterns.to_vec()
+        };
```

Comment:
Actually, I was just thinking about this more and if someone uses backslashes in a config file then it won't work on linux. I'm going to revert this part of the change in order to force people to use forward slashses.

Reply:
I'll try to think about a cross-platform solution to that... I'll start by looking into what `git` does for *.gitignore*.

Reply:
So, without this conversion, Windows paths are still broken.

```shell
C:\...\dprint>target\release\dprint.exe check crates\cli-core\src\*.rs --verbose
[VERBOSE]: Creating directory: C:\Users\Roy\AppData\Local\dprint\cache
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\cache-manifest.json
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugin-cache-manifest.json
[VERBOSE]: Checking path exists: ./.dprintrc.json
[VERBOSE]: Reading file: ./.dprintrc.json
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-markdown\dprint-plugin-markdown-0.5.1.cached
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-json\dprint-plugin-json-0.8.0.cached
[VERBOSE]: Checking path exists: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-prettier\0.2.1\dprint-plugin-prettier.exe
[VERBOSE]: Reading file: C:\Users\Roy\AppData\Local\dprint\cache\plugins\dprint-plugin-typescript\dprint-plugin-typescript-0.40.3.cached
[VERBOSE]: Globbing: ["crates\\cli-core\\src\\checksums.rs", "crates\\cli-core\\src\\lib.rs", "crates\\cli-core\\src\\terminal.rs", "crates\\cli-core\\src\\types.rs", "crates\\cli-core\\src\\url_utils.rs", "!website/playground/build", "!scripts/build-website", "!**/node_modules", "!**/dist", "!**/target", "!**/*-lock.json"]
[VERBOSE]: File(s) matched: []
[VERBOSE]: Finished globbing in 9ms
No files found to format with the specified plugins. You may want to try using `dprint output-file-paths` to see which files it's finding.
```

Maybe include it and note that the '/' *nix-style seperator is preferred for cross-platform use?